core/services/ocr2/plugins/ocr2keeper/evm21/logprovider/log_packer.go

```diff
@@ -28,7 +28,7 @@ func (p *logEventsPacker) PackLogData(log logpoller.Log) ([]byte, error) {
 	}
 	b, err := p.abi.Pack("_log", &automation_utils_2_1.Log{
 		Index:       big.NewInt(log.LogIndex),
-		TxIndex:     big.NewInt(0), // Exploratory: Add this to the logpoller and pass correct value
+		Timestamp:   big.NewInt(log.BlockTimestamp.Unix()),
```

Comment:
Keep the comment for TxIndex

Reply:
i think there woudln't be any txIndex anymore